feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.html

```diff
@@ -0,0 +1,38 @@
+<label for="cartNumber"
+  >{{ 'asm.assignCart.cartNumber' | cxTranslate }}
+  <div
+    role="search"
+    aria-label="Assign a cart id to customer"
+    class="assignCart"
+    [class.active]="cartId.valid"
+    (click)="cartIdElement.focus()"
+  >
+    <input
+      autocomplete="off"
+      #cartIdElement
+      formcontrolname="cartNumber"
+      [formControl]="cartId"
+      (keydown.enter)="assignCartToCustomer()"
+      aria-label="Enter cart id"
+    />
+    <button
+      class="reset"
+      aria-label="Reset"
+      [class.visible]="cartId.value?.length > 0"
+      (click)="clearText()"
+    >
+      <cx-icon class="cx-icon fas fa-times-circle"></cx-icon>
+    </button>
+    <div role="presentation" class="search-icon">
+      <cx-icon class="cx-icon fas fa-search"></cx-icon>
+    </div>
+  </div>
+  <button
```

Comment:
When the network is slower or the backend takes some time to process the request, there is no work in progress feedback for the user.  This probably should be handled in some way.

Reply:
The most common UI mechanism to provide quick feedback are spinners that show that processing is happening.
You can look at the ASM customer search, it has a three dot style spinner.
Most of the storefront UI uses a circular spinner.  

The other purpose for the form to have processing state is to prevent multiple submits of the same data.  The form should not be submittable the the post is processing.